client/me/help/help-contact/index.jsx

```diff
@@ -256,15 +257,59 @@ class HelpContact extends React.Component {
 	};
 
 	submitSupportForumsTopic = ( contactForm ) => {
-		const { subject, message } = contactForm;
-		const { currentUserLocale } = this.props;
+		const {
+			helpSiteIsJetpack,
+			helpSiteIsNotWpCom,
+			site,
+			subject,
+			message,
+			userDeclaresNoSite,
+			userDeclaredUrl,
+			userRequestsHidingUrl,
+		} = contactForm;
+		const { currentUserLocale, translate } = this.props;
 
 		this.setState( { isSubmitting: true } );
 		this.recordCompactSubmit( 'forums' );
 
+		let blogHelpMessage = translate( "I don't have a site linked to this WordPress.com account" );
+
+		if ( userDeclaresNoSite ) {
+			blogHelpMessage = translate( "I don't have a site with WordPress.com yet" );
+		}
+
+		if ( site || userDeclaredUrl ) {
+			const siteUrl = userDeclaredUrl
+				? withoutHttp( userDeclaredUrl.trim() )
+				: withoutHttp( site.URL );
+
+			blogHelpMessage = translate( 'The site I need help with is %(siteUrl)s.', {
+				args: {
+					siteUrl: userRequestsHidingUrl
+						? translate( '[visible only to staff]' ) + ' help@' + siteUrl
+						: siteUrl,
+				},
+			} );
+
+			if ( helpSiteIsNotWpCom ) {
+				const jetpackMessage = helpSiteIsJetpack
+					? translate( 'It is not hosted by WordPress.com, but it is connected with Jetpack.' )
+					: translate( 'It is not hosted by WordPress.com or connected with Jetpack.' );
+
+				blogHelpMessage += ' ' + jetpackMessage;
+			}
+
+			if ( userDeclaredUrl ) {
+				blogHelpMessage +=
+					' ' + translate( 'This site is not linked to my WordPress.com account.' );
+			}
+		}
+
+		const forumMessage = message + '\n\n' + blogHelpMessage;
```

Comment:
In other support channels, e.g., when submitting a ticked with `submitKayakoTicket`, we add the "meta info" at the beginning of the message, not at the end. Let's make the message formats adhere to some convention.

Reply:
I'm happy to update that, but my initial reasoning was that the "meta info" already appears at the end for threads created directly in the Forums. For example, you can see that on this thread: https://wordpress.com/forums/topic/home-page-not-displaying-menu/

Because of that, I feel like changing it around might be more inconsistent or slightly jarring for volunteers. Let me know if you still think it's worth switching around though! :)

Reply:
I just wanted to point out that other support methods format the message differently. I don't really know what's right for this situation. Maybe @KokkieH and @supernovia will have more qualified opinions about this.

Reply:
This is for the "The blog I need help with is..." section, correct?

In the forums that has always been appended at the end of the first message, both for threads created directly in the forums, and for threads created via the contact form before it stopped happening without any apparent explanation two years ago, as reported in #38280